src/PocketMine.php

```diff
@@ -175,9 +175,6 @@ function emit_performance_warnings(\Logger $logger){
 		if(((int) ini_get('zend.assertions')) !== -1){
 			$logger->warning("Debugging assertions are enabled. This may degrade performance. To disable them, set `zend.assertions = -1` in php.ini.");
 		}
-		if(\Phar::running(true) === ""){
-			$logger->warning("Non-packaged installation detected. This will degrade autoloading speed and make startup times longer.");
```

Comment:
rip phar

Reply:
To be honest I think this was always a lie. Decompressing the phar contents negates any gain had by packing everything into one file. The only way this warning would have been valid is if the phar had zero compression.

Reply:
It was probably manufactured during the time where releases only had function imports.

Reply:
I'd forgotten the phar releases used to be preprocessed. You might be right about that.

Reply:
I thought windows had better performance loading from a single archive.

Reply:
Like I said, the I/O overhead is offset by decompression overhead. There's little observable difference in practice.

On *nix src is pretty much always going to be faster, and that's where the majority of the userbase is.